tests/raftstore/test_conf_change.rs

```diff
@@ -416,7 +416,7 @@ fn test_split_brain<T: Simulator>(cluster: &mut Cluster<T>) {
     pd_client.must_add_peer(r1, new_peer(3, 3));
 
     // leader isolation
-    cluster.must_transfer_leader(r1, new_peer(1, 1));
+    cluster.must_transfer_leader(r1, new_peer(2, 2));
```

Comment:
The isolated peer should not be the old leader, otherwise the new leader will dectect it's not healthy, and rejects adding new peer.